src/lib/components/ModalLayout.svelte

```diff
@@ -1,13 +1,15 @@
 <script lang="ts">
 	import * as modal from '$lib/utils/modal';
 
+	const modalStore = modal.store;
+
+	$: store = $modalStore;
+
 	const clickOutside = () => {
 		modal.hide();
 	};
 
-	const modalStore = modal.store;
-	// Hack to make svelte typecheck in the markup section.
-	$: store = $modalStore;
+	// TODO Hack to make svelte typecheck in the markup section.
```

Comment:
This is not a `//TODO`. Declaring the store and then referencing was the hack to circumvent a type error. If you're removing it we can remove the comment.